netbox/dcim/models.py

```diff
@@ -553,9 +559,15 @@ class Rack(ChangeLoggedModel, CustomFieldModel):
 
     class Meta:
         ordering = ['site', 'group', 'name']
-        unique_together = [
-            ['group', 'name'],
-            ['group', 'facility_id'],
+        constraints = [
+            models.UniqueConstraint(
+                fields=['group', 'name'],
+                name='rack_group_name_key'
```

Comment:
Wrong table name for Rack

Reply:
It’s actually `rack`, `group`, `name`, not `rack_group`, `name`. It’s annoying that this is so ambiguous, which is why I opted to remove the `_`. Do you have any preferences how to deal with this then?